.github/workflows/cd.yml

```
@@ -49,4 +49,4 @@ jobs:
 
             # 새로운 컨테이너 실행
             echo "Running new container $CONTAINER_NAME"
-            docker run -d --name "$CONTAINER_NAME" -p 8080:8080 118497224929.dkr.ecr.ap-northeast-2.amazonaws.com/clip-trip/server:latest
+            docker run -d --name "$CONTAINER_NAME" -p 8080:8080 118497224929.dkr.ecr.ap-northeast-2.amazonaws.com/clip-trip/server:latest
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)